src/datreant/core/collections.py

```diff
@@ -439,6 +439,17 @@ def relpaths(self):
         return [member.relpath for member in self]
 
     @property
+    def parents(self):
```

Comment:
The aggregation part is important for me for my understanding. I think I'm confused by the naming. I will think a bit more about that and let you know in the following days. Otherwise yeah parents makes sense with the current functionality as a aggregation class.